lua/nvim-treesitter.lua

```diff
@@ -48,15 +49,16 @@ function M.statusline(opts)
     return
   end
   local options = opts or {}
-  if type(opts) == "number" then
-    options = { indicator_size = opts }
-  end
+  -- if type(opts) == "number" then
+  --   options = { indicator_size = opts }
+  -- end
+  local bufnr = options.bufnr or 0
   local indicator_size = options.indicator_size or 100
   local type_patterns = options.type_patterns or { "class", "function", "method" }
   local transform_fn = options.transform_fn or transform_line
   local separator = options.separator or " -> "
 
-  local current_node = ts_query.get_node_at_cursor()
+  local current_node = ts_utils.get_node_at_cursor()
```

Comment:
oops...

Reply:
I could find the `get_node_text()` upstream but I just cant find the `get_node_at_cursor() `upstream. I am on the latest build of nvim

Reply:
Yeah, that was my bad. I got a bit carried away there. (This _should_ be upstreamed, but clearly hasn't yet.)